OasisDEX/oasis-borrow
# Ades/risk slider

# [Configurable slider](https://app.shortcut.com/oazo-apps/story/6423/ui-ltv-slider)
  
## Changes 👷‍♀️
- inject config into risk adjustment slider, and inject component in strategy config
- co-locates strategy configs with the view components.
- fixes the step numbers on the aave open flow
  
## How to test 🧪
- adjust risk slider on both multiply and earn products - the boundaries should work correctly, and the position should get creates as before.
- ensure flow numbers with/without creating a proxy in the open flow make sense
